### ISSUE TITLE
Acceptance tests fail

### ISSUE THREAD

**Issue by b5white** [unclebob/fitnesse#898]
Running an individual test, like
FitNesse.SuiteAcceptanceTests.SuiteAuthenticationTests.AlwaysSecureOperation
succeeds.

Running a lower level suite, like
FitNesse.SuiteAcceptanceTests.SuiteAuthenticationTests
succeeds.

But running the entire suite, FitNesse.SuiteAcceptanceTests, throws exceptions in every test.
I get "0 right, 0 wrong, x exceptions" in most tests.

Most of the exceptions are in teardown.
 java.lang.NullPointerException

**Comment by b5white** [comment]
tl;dr  NullPointerException with context from log.

&lt;br/&gt;&lt;table&gt;
	&lt;tr class="slimRowTitle"&gt;
		&lt;td&gt;Library&lt;/td&gt;
	&lt;/tr&gt;
	&lt;tr class="slimRowColor0"&gt;
		&lt;td&gt;&lt;span class="pass"&gt;page driver&lt;/span&gt;&lt;/td&gt;
	&lt;/tr&gt;
	&lt;tr class="slimRowColor1"&gt;
		&lt;td&gt;&lt;span class="pass"&gt;echo fixture&lt;/span&gt;&lt;/td&gt;
	&lt;/tr&gt;
&lt;/table&gt;
&lt;br/&gt;&lt;/div&gt;
&lt;/div&gt;
&lt;br/&gt;When viewing versions, the user who created each &lt;i&gt;previous&lt;/i&gt; version is listed along with each version.  Anonymous users are omitted.&lt;br/&gt;&lt;br/&gt;&lt;table&gt;
	&lt;tr class="slimRowTitle"&gt;
		&lt;td colspan="4"&gt;script&lt;/td&gt;
	&lt;/tr&gt;
	&lt;tr class="scenario closed"&gt;
		&lt;td class="error"&gt;save page&lt;/td&gt;
		&lt;td class="error"&gt;MyPage&lt;/td&gt;
		&lt;td class="error"&gt;by user&lt;/td&gt;
		&lt;td class="error"&gt;Bob&lt;/td&gt;
	&lt;/tr&gt;&lt;tr class="scenario-detail closed-detail"&gt;&lt;td colspan="4"&gt;&lt;table&gt;
	&lt;tr class="slimRowTitle"&gt;
		&lt;td&gt;scenario&lt;/td&gt;
		&lt;td&gt;save page&lt;/td&gt;
		&lt;td&gt;page&lt;/td&gt;
		&lt;td&gt;by user&lt;/td&gt;
		&lt;td colspan="6"&gt;user&lt;/td&gt;
	&lt;/tr&gt;
	&lt;tr class="exception closed"&gt;
		&lt;td class="error"&gt;check&lt;/td&gt;
		&lt;td class="error"&gt;request page save&lt;/td&gt;
		&lt;td class="error"&gt;MyPage&lt;/td&gt;
		&lt;td class="error"&gt;with contents&lt;/td&gt;
		&lt;td class="error"&gt;empty&lt;/td&gt;
		&lt;td class="error"&gt;by user&lt;/td&gt;
		&lt;td class="error"&gt;Bob&lt;/td&gt;
		&lt;td class="error"&gt;and password&lt;/td&gt;
		&lt;td class="error"&gt;empty&lt;/td&gt;
		&lt;td class="error"&gt;303&lt;/td&gt;
	&lt;/tr&gt;&lt;tr class="exception-detail closed-detail"&gt;&lt;td colspan="10"&gt;&lt;pre&gt;__EXCEPTION__:java.lang.NullPointerException
	at fitnesse.fixtures.PageDriver.requestPageSaveWithContentsByUserAndPassword(PageDriver.java:71) [fitnesse-20151230-standalone.jar]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) [rt.jar:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at java.lang.reflect.Method.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at fitnesse.slim.fixtureInteraction.DefaultInteraction.methodInvoke(DefaultInteraction.java:81) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.callMethod(MethodExecutor.java:57) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.invokeMethod(MethodExecutor.java:38) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.findAndInvoke(MethodExecutor.java:72) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.LibraryMethodExecutor.execute(LibraryMethodExecutor.java:18) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.StatementExecutor.getMethodExecutionResult(StatementExecutor.java:135) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.StatementExecutor.call(StatementExecutor.java:108) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.instructions.CallInstruction.executeInternal(CallInstruction.java:35) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.instructions.Instruction.execute(Instruction.java:29) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor$Executive.executeStatement(ListExecutor.java:49) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor$Executive.executeStatements(ListExecutor.java:43) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor.execute(ListExecutor.java:85) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.executeInstructions(SlimServer.java:84) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.processOneSetOfInstructions(SlimServer.java:77) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.tryProcessInstructions(SlimServer.java:64) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.serve(SlimServer.java:49) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.handle(SlimService.java:154) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.acceptOne(SlimService.java:162) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.accept(SlimService.java:124) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.startWithFactory(SlimService.java:76) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.main(SlimService.java:56) [fitnesse-20151230-standalone.jar]&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;
	&lt;tr class="slimRowColor4"&gt;
		&lt;td&gt;$IT&amp;lt;-[MyPage]&lt;/td&gt;
		&lt;td&gt;echo&lt;/td&gt;
		&lt;td colspan="8"&gt;MyPage&lt;/td&gt;
	&lt;/tr&gt;
&lt;/table&gt;&lt;/td&gt;&lt;/tr&gt;
	&lt;tr class="scenario closed"&gt;
		&lt;td class="error"&gt;save page&lt;/td&gt;
		&lt;td class="error"&gt;MyPage&lt;/td&gt;
		&lt;td class="error"&gt;by user&lt;/td&gt;
		&lt;td class="error"&gt;Bill&lt;/td&gt;
	&lt;/tr&gt;&lt;tr class="scenario-detail closed-detail"&gt;&lt;td colspan="4"&gt;&lt;table&gt;
	&lt;tr class="slimRowTitle"&gt;
		&lt;td&gt;scenario&lt;/td&gt;
		&lt;td&gt;save page&lt;/td&gt;
		&lt;td&gt;page&lt;/td&gt;
		&lt;td&gt;by user&lt;/td&gt;
		&lt;td colspan="6"&gt;user&lt;/td&gt;
	&lt;/tr&gt;
	&lt;tr class="exception closed"&gt;
		&lt;td class="error"&gt;check&lt;/td&gt;
		&lt;td class="error"&gt;request page save&lt;/td&gt;
		&lt;td class="error"&gt;MyPage&lt;/td&gt;
		&lt;td class="error"&gt;with contents&lt;/td&gt;
		&lt;td class="error"&gt;empty&lt;/td&gt;
		&lt;td class="error"&gt;by user&lt;/td&gt;
		&lt;td class="error"&gt;Bill&lt;/td&gt;
		&lt;td class="error"&gt;and password&lt;/td&gt;
		&lt;td class="error"&gt;empty&lt;/td&gt;
		&lt;td class="error"&gt;303&lt;/td&gt;
	&lt;/tr&gt;&lt;tr class="exception-detail closed-detail"&gt;&lt;td colspan="10"&gt;&lt;pre&gt;__EXCEPTION__:java.lang.NullPointerException
	at fitnesse.fixtures.PageDriver.requestPageSaveWithContentsByUserAndPassword(PageDriver.java:71) [fitnesse-20151230-standalone.jar]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) [rt.jar:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at java.lang.reflect.Method.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at fitnesse.slim.fixtureInteraction.DefaultInteraction.methodInvoke(DefaultInteraction.java:81) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.callMethod(MethodExecutor.java:57) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.invokeMethod(MethodExecutor.java:38) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.findAndInvoke(MethodExecutor.java:72) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.LibraryMethodExecutor.execute(LibraryMethodExecutor.java:18) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.StatementExecutor.getMethodExecutionResult(StatementExecutor.java:135) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.StatementExecutor.call(StatementExecutor.java:108) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.instructions.CallInstruction.executeInternal(CallInstruction.java:35) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.instructions.Instruction.execute(Instruction.java:29) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor$Executive.executeStatement(ListExecutor.java:49) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor$Executive.executeStatements(ListExecutor.java:43) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor.execute(ListExecutor.java:85) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.executeInstructions(SlimServer.java:84) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.processOneSetOfInstructions(SlimServer.java:77) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.tryProcessInstructions(SlimServer.java:64) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.serve(SlimServer.java:49) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.handle(SlimService.java:154) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.acceptOne(SlimService.java:162) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.accept(SlimService.java:124) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.startWithFactory(SlimService.java:76) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.main(SlimService.java:56) [fitnesse-20151230-standalone.jar]&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;
	&lt;tr class="slimRowColor4"&gt;
		&lt;td&gt;$IT&amp;lt;-[MyPage]&lt;/td&gt;
		&lt;td&gt;echo&lt;/td&gt;
		&lt;td colspan="8"&gt;MyPage&lt;/td&gt;
	&lt;/tr&gt;
&lt;/table&gt;&lt;/td&gt;&lt;/tr&gt;
	&lt;tr class="scenario closed"&gt;
		&lt;td class="error"&gt;it's versions should contain&lt;/td&gt;
		&lt;td colspan="3" class="error"&gt;Bob&lt;/td&gt;
	&lt;/tr&gt;&lt;tr class="scenario-detail closed-detail"&gt;&lt;td colspan="4"&gt;&lt;table&gt;
	&lt;tr class="slimRowTitle"&gt;
		&lt;td&gt;scenario&lt;/td&gt;
		&lt;td&gt;it's versions should contain&lt;/td&gt;
		&lt;td colspan="2"&gt;content&lt;/td&gt;
	&lt;/tr&gt;
	&lt;tr class="scenario closed"&gt;
		&lt;td class="error"&gt;page&lt;/td&gt;
		&lt;td class="error"&gt;$IT-&gt;[MyPage]?versions&lt;/td&gt;
		&lt;td class="error"&gt;should contain&lt;/td&gt;
		&lt;td class="error"&gt;Bob&lt;/td&gt;
	&lt;/tr&gt;&lt;tr class="scenario-detail closed-detail"&gt;&lt;td colspan="4"&gt;&lt;table&gt;
	&lt;tr class="slimRowTitle"&gt;
		&lt;td&gt;scenario&lt;/td&gt;
		&lt;td&gt;page&lt;/td&gt;
		&lt;td&gt;source&lt;/td&gt;
		&lt;td&gt;should contain&lt;/td&gt;
		&lt;td&gt;text&lt;/td&gt;
	&lt;/tr&gt;
	&lt;tr class="exception closed"&gt;
		&lt;td class="error"&gt;check&lt;/td&gt;
		&lt;td class="error"&gt;request page&lt;/td&gt;
		&lt;td class="error"&gt;$IT-&gt;[MyPage]?versions&lt;/td&gt;
		&lt;td colspan="2" class="error"&gt;200&lt;/td&gt;
	&lt;/tr&gt;&lt;tr class="exception-detail closed-detail"&gt;&lt;td colspan="5"&gt;&lt;pre&gt;__EXCEPTION__:java.lang.NullPointerException
	at fitnesse.fixtures.ResponseRequester.execute(ResponseRequester.java:37) [fitnesse-20151230-standalone.jar]
	at fitnesse.fixtures.PageDriver.requestPage(PageDriver.java:51) [fitnesse-20151230-standalone.jar]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) [rt.jar:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at java.lang.reflect.Method.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at fitnesse.slim.fixtureInteraction.DefaultInteraction.methodInvoke(DefaultInteraction.java:81) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.callMethod(MethodExecutor.java:57) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.invokeMethod(MethodExecutor.java:38) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.findAndInvoke(MethodExecutor.java:72) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.LibraryMethodExecutor.execute(LibraryMethodExecutor.java:18) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.StatementExecutor.getMethodExecutionResult(StatementExecutor.java:135) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.StatementExecutor.call(StatementExecutor.java:108) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.instructions.CallInstruction.executeInternal(CallInstruction.java:35) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.instructions.Instruction.execute(Instruction.java:29) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor$Executive.executeStatement(ListExecutor.java:49) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor$Executive.executeStatements(ListExecutor.java:43) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor.execute(ListExecutor.java:85) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.executeInstructions(SlimServer.java:84) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.processOneSetOfInstructions(SlimServer.java:77) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.tryProcessInstructions(SlimServer.java:64) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.serve(SlimServer.java:49) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.handle(SlimService.java:154) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.acceptOne(SlimService.java:162) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.accept(SlimService.java:124) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.startWithFactory(SlimService.java:76) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.main(SlimService.java:56) [fitnesse-20151230-standalone.jar]&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;
	&lt;tr class="exception closed"&gt;
		&lt;td class="error"&gt;ensure&lt;/td&gt;
		&lt;td class="error"&gt;content contains&lt;/td&gt;
		&lt;td colspan="3" class="error"&gt;Bob&lt;/td&gt;
	&lt;/tr&gt;&lt;tr class="exception-detail closed-detail"&gt;&lt;td colspan="5"&gt;&lt;pre&gt;__EXCEPTION__:java.lang.NullPointerException
	at fitnesse.fixtures.ResponseExaminer.extractValueFromResponse(ResponseExaminer.java:80) [fitnesse-20151230-standalone.jar]
	at fitnesse.fixtures.PageDriver.contentContains(PageDriver.java:130) [fitnesse-20151230-standalone.jar]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) [rt.jar:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at java.lang.reflect.Method.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at fitnesse.slim.fixtureInteraction.DefaultInteraction.methodInvoke(DefaultInteraction.java:81) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.callMethod(MethodExecutor.java:57) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.invokeMethod(MethodExecutor.java:38) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.findAndInvoke(MethodExecutor.java:72) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.LibraryMethodExecutor.execute(LibraryMethodExecutor.java:18) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.StatementExecutor.getMethodExecutionResult(StatementExecutor.java:135) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.StatementExecutor.call(StatementExecutor.java:108) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.instructions.CallInstruction.executeInternal(CallInstruction.java:35) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.instructions.Instruction.execute(Instruction.java:29) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor$Executive.executeStatement(ListExecutor.java:49) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor$Executive.executeStatements(ListExecutor.java:43) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor.execute(ListExecutor.java:85) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.executeInstructions(SlimServer.java:84) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.processOneSetOfInstructions(SlimServer.java:77) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.tryProcessInstructions(SlimServer.java:64) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.serve(SlimServer.java:49) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.handle(SlimService.java:154) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.acceptOne(SlimService.java:162) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.accept(SlimService.java:124) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.startWithFactory(SlimService.java:76) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.main(SlimService.java:56) [fitnesse-20151230-standalone.jar]&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;
	&lt;tr class="exception closed"&gt;
		&lt;td class="error"&gt;show&lt;/td&gt;
		&lt;td colspan="4" class="error"&gt;content&lt;/td&gt;
	&lt;/tr&gt;&lt;tr class="exception-detail closed-detail"&gt;&lt;td colspan="5"&gt;&lt;pre&gt;__EXCEPTION__:java.lang.NullPointerException
	at fitnesse.fixtures.ResponseRequester.html(ResponseRequester.java:54) [fitnesse-20151230-standalone.jar]
	at fitnesse.fixtures.PageDriver.content(PageDriver.java:157) [fitnesse-20151230-standalone.jar]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) [rt.jar:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at java.lang.reflect.Method.invoke(Unknown Source) [rt.jar:1.8.0_51]
	at fitnesse.slim.fixtureInteraction.DefaultInteraction.methodInvoke(DefaultInteraction.java:81) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.callMethod(MethodExecutor.java:57) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.invokeMethod(MethodExecutor.java:38) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.MethodExecutor.findAndInvoke(MethodExecutor.java:72) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.LibraryMethodExecutor.execute(LibraryMethodExecutor.java:18) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.StatementExecutor.getMethodExecutionResult(StatementExecutor.java:135) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.StatementExecutor.call(StatementExecutor.java:108) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.instructions.CallInstruction.executeInternal(CallInstruction.java:35) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.instructions.Instruction.execute(Instruction.java:29) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor$Executive.executeStatement(ListExecutor.java:49) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor$Executive.executeStatements(ListExecutor.java:43) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.ListExecutor.execute(ListExecutor.java:85) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.executeInstructions(SlimServer.java:84) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.processOneSetOfInstructions(SlimServer.java:77) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.tryProcessInstructions(SlimServer.java:64) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimServer.serve(SlimServer.java:49) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.handle(SlimService.java:154) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.acceptOne(SlimService.java:162) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.accept(SlimService.java:124) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.startWithFactory(SlimService.java:76) [fitnesse-20151230-standalone.jar]
	at fitnesse.slim.SlimService.main(SlimService.java:56) [fitnesse-20151230-standalone.jar]&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;/table&gt;&lt;/td&gt;&lt;/tr&gt;

**Comment by b5white** [comment]
But I only get this when it is run as part of the suite. Run by itself or with just its parent suite works fine.

**Comment by b5white** [comment]
I'm going to let this go.
As someone else pointed out, this only affects the java tests and the .NT tests all run fine.
It seems like an issue with the java test runner, so I should be unaffected.
